2016-03-30 20:18:02	Type:Debug	File:QAFCore.cpp	Line:70	Function:void __thiscall QAF::QAFCore::initCore(void)
	"正在加载核心模块：QAF::ConfigSystem"
2016-03-30 20:18:02	Type:Debug	File:QAFCore.cpp	Line:70	Function:void __thiscall QAF::QAFCore::initCore(void)
	"正在加载核心模块：QAF::ObjectSystem"
2016-03-30 20:18:02	Type:Debug	File:QAFCore.cpp	Line:90	Function:void __thiscall QAF::QAFCore::initExtent(void)
	"正在加载扩展模块：QAF::PluginSystem"
2016-03-30 20:18:02	Type:Debug	File:PluginSystem.cpp	Line:42	Function:void __thiscall QAF::PluginSystem::install(void)
	"正在加载插件:" "zxj" "test for plugin" 1 "1.0"
2016-03-30 20:18:02	Type:Debug	File:PluginSystem.cpp	Line:42	Function:void __thiscall QAF::PluginSystem::install(void)
	"正在加载插件:" "zxj" "UI plugins" 1 "1.0"
2016-03-30 20:18:14	Type:Debug	File:QAFCore.cpp	Line:33	Function:__thiscall QAF::QAFCore::~QAFCore(void)
	"正在卸载核心模块：QAF::ConfigSystem"
2016-03-30 20:18:14	Type:Debug	File:QAFCore.cpp	Line:33	Function:__thiscall QAF::QAFCore::~QAFCore(void)
	"正在卸载核心模块：QAF::ObjectSystem"
2016-03-30 20:18:14	Type:Debug	File:QAFCore.cpp	Line:33	Function:__thiscall QAF::QAFCore::~QAFCore(void)
	"正在卸载扩展模块：QAF::PluginSystem"
2016-03-30 20:18:14	Type:Fatal	File:global\qglobal.cpp	Line:2974	Function:void __cdecl qt_assert_x(const char *,const char *,const char *,int) throw()
	ASSERT failure in QCoreApplication::sendEvent: "Cannot send events to objects owned by a different thread. Current thread 10027e8. Receiver '' (of type 'QAF::QAFCore') was created in thread fd5da8", file kernel\qcoreapplication.cpp, line 553
